test_segmentation_model.py

```diff
@@ -60,7 +61,14 @@ def test(gcp_bucket, dataset_id, model_id, batch_size):
         train_config['optimizer'],
         Path(local_model_dir, model_id, "model.hdf5").as_posix())
 
-    results = compiled_model.evaluate_generator(test_generator)
+    # manually check metric memory location and inheritance chain
```

Comment:
this comment in the code is likely going to be real confusing 6 months from now :) -- so I'd say either we just remove it and possibly move it to some sort of documentation or uncomment it and include something like a verbose flag

Reply:
removed